Makefile

```diff
@@ -591,10 +591,10 @@ endif
 
 gen-serialized: ## Generates serialization methods for hot structs
 	# This tool only works at a file level, not at a package level.
```

Comment:
I'd suggest adding a line to the effect that:
> There will be some warnings about "unresolved identifiers", but that is because of the above problem. Since we are generating methods for all the relevant files at a package level, all identifiers will be resolved. An alternative to remove the warnings would be to temporarily move all the structs to the same file, but that involves a lot of manual work.

to preserve the context.